libs/designsystem/src/lib/components/card/card.component.html

```diff
@@ -1,5 +1,10 @@
-<ng-content select="kirby-card-header"></ng-content>
-<div class="content-wrapper" [class.padding]="hasPadding">
-  <ng-content></ng-content>
-</div>
-<ng-content select="kirby-card-footer"></ng-content>
+<ng-container>
```

Comment:
Does this `ng-container` have any use when we are not adding a structural directive or similar?